docs/docs/examples/cat-videos.mdx

```diff
@@ -0,0 +1,156 @@
+privilege modeled in this example is "view access." Every owner has view access
+to their objects, and this privilege can be granted to other users as well. The
+Ory Keto application interprets the special `*` user ID as any user, including
+anonymous users. Note that Ory Keto does not interpret this subject any
+differently from other subjects. It also does not know anything about directory
+structures or induced ownership.
+
+## Starting the Example
+
+First, [install Keto](../install).
+
+Now you can start the example using either `docker-compose` or a bash script.
+The bash script requires you to have the `keto` binary in your `$PATH` as
+follows:
+
+```shell
+$ sudo mv ./keto /usr/local/bin/
+$ keto help
+```
+
+Alternatively, using Docker automatically gets the required images:
+
+```shell
+# clone the repository if you don't have it yet
+git clone git@github.com:ory/keto.git && cd keto
+
+docker-compose -f contrib/cat-videos-example/docker-compose.yml up
+# or
+./contrib/cat-videos-example/up.sh
+
+# output: all initially created relation tuples
+
+# NAMESPACE       OBJECT          RELATION NAME   SUBJECT
+# videos          /cats/1.mp4     owner           videos:/cats#owner
+# videos          /cats/1.mp4     view            videos:/cats/1.mp4#owner
+# videos          /cats/1.mp4     view            *
+# videos          /cats/2.mp4     owner           videos:/cats#owner
+# videos          /cats/2.mp4     view            videos:/cats/2.mp4#owner
+# videos          /cats           owner           cat lady
+# videos          /cats           view            videos:/cats#owner
+```
+
+## State of the System
+
+At the current state only one user with the username `cat lady` has added
+videos. Both videos are in the `/cats` directory owned by `cat lady`. The file
+`/cats/1.mp4` can be viewed by anyone (`*`), while `/cats/2.mp4` has no extra
+sharing options, and can therefore only be viewed by its owner, `cat lady`. The
+relation tuple definitions are located in the
+`contrib/cat-videos-example/relation-tuples` directory.
+
+## Simulating the Client
+
+Now you can open a second terminal to run the queries against, just like the
+video service client would do. In this example we will use the Keto CLI client.
+
+:::info
+
+If you want to run the Keto CLI within **Docker**, set the alias
+
+```shell
+alias keto="docker run -it --network cat-videos-example_default -e KETO_GRPC_URL=\"keto:4467\" oryd/keto:latest"
+```
+
+in your terminal session.
+
+:::
+
+Set the remote endpoint so that the Keto CLI knows where to connect to (not
+necessary if using Docker):
+
+```shell
+export KETO_GRPC_URL="127.0.0.1:4467"
+```
+
+First off, we get a request by an anonymous user that would like to view
+`/cats/2.mp4`. The client now has to ask Keto if that operation should be
+allowed or denied.
+
+```shell
+# Is "*" allowed to "view" the object "videos":"/cats/2.mp4"?
+keto check "*" view videos /cats/2.mp4
+# output:
+
+# false
```

Comment:
```suggestion
$ keto check "*" view videos /cats/2.mp4

Denied
```

Reply:
I hate docs that have commands defined in a way that I cannot copy-paste them. Syntax highlighting also makes no sense with this format. Are you sure that we want to have them like that?

Reply:
The problem is that it's not clear what's input and what's output otherwise. Syntax highlighting also works like this if you have `shell-session` as code type. But I agree that it sucks to copy this. However, if it's singular commands, I think it's ok